diff --git a/node_modules/@google/gemini-cli-a2a-server/dist/a2a-server.mjs b/node_modules/@google/gemini-cli-a2a-server/dist/a2a-server.mjs
index ade3d6e..01616e0 100755
--- a/node_modules/@google/gemini-cli-a2a-server/dist/a2a-server.mjs
+++ b/node_modules/@google/gemini-cli-a2a-server/dist/a2a-server.mjs
@@ -387410,7 +387410,7 @@ var init_fallbackStrategy = __esm({
     init_models();
     FallbackStrategy = class {
       name = "fallback";
-      async route(context2, config3, _baseLlmClient, _localLiteRtLmClient) {
+      async route(context2, config3, _baseLlmClient, _localLiteRtLmClient) { return null; 
         const requestedModel = context2.requestedModel ?? config3.getModel();
         const resolvedModel = resolveModel(requestedModel, config3.getGemini31LaunchedSync?.() ?? false);
         const service = config3.getModelAvailabilityService();
@@ -387443,7 +387443,7 @@ var init_overrideStrategy = __esm({
     init_models();
     OverrideStrategy = class {
       name = "override";
-      async route(context2, config3, _baseLlmClient, _localLiteRtLmClient) {
+      async route(context2, config3, _baseLlmClient, _localLiteRtLmClient) { return null; 
         const overrideModel = context2.requestedModel ?? config3.getModel();
         if (isAutoModel(overrideModel, config3)) {
           return null;
